rootfs/api/models/config.py

```diff
@@ -53,35 +54,6 @@ def healthcheck(self):
             'failure_threshold': failure_threshold,
         }
 
-    def set_healthchecks(self):
-        """Defines default values for HTTP healthchecks"""
-        if not {k: v for k, v in self.values.items() if k.startswith('HEALTHCHECK_')}:
-            return
-
-        # fetch set health values and any defaults
-        # this approach allows new health items to be added without issues
-        health = self.healthcheck()
-        if not health:
-            return
-
-        # HTTP GET related
-        self.values['HEALTHCHECK_URL'] = health['path']
-
-        # Number of seconds after which the probe times out.
```

Comment:
Can you move some of these comments (and mostly the links) to the probe schema you have